clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ApiConstants.scala

```diff
@@ -8,7 +8,7 @@ object ApiConstants {
   def forceString: String = "force"
   def apiString: String = "api"
   def queryString: String = "query"
-  def queryAllString: String = "queryall"
+  def rawQueryString: String = "rawquery"
```

Comment:
Ergh, just occurred to me that ripping this out will break the backfill (it queries with include-deleted=true). We should probably keep the web endpoints until we get the chance to bump the version of the client used there.